schemas/EGA.analysis.json

```diff
@@ -5,9 +5,190 @@
+        "uniqueItems": true,
+        "additionalProperties": false,
+        "items": {
+          "title": "Experimental design item",
+          "description": "One of the experimental designs of the analysis.",
+          "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/experimental_design"
+        }
+      },
+
+      "types_of_data": {
+        "type": "array",
+        "title": "Type of data [IAO:0000100]",
+        "description": "Type of raw data the analysis uses to obtain the processed files.",
+        "additionalProperties": false,
+        "uniqueItems": true,
+        "minItems": 1,
+        "items": {
+          "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/types_of_raw_data"
+        }
+      },
+
+      "analysis_type_specifications": {
+        "type": "object",
+        "title": "Analysis type specifications",
+        "description": "Node containing different sets of fields that depend on the specific analysis type. Depending on the analysis types different metadata will be required.",
+        "additionalProperties": false,
+        "required": ["analysis_types"],
+        "properties": {
+          "analysis_types": {
+            "type": "array",
+            "title": "List of analysis types",
+            "description": "Array of all analysis types applicable to this analysis. Details on how the analysis was performed (instruments, software, procedure...) shall be included in the 'analysis_protocols' field, not here. For example, if the analysis includes sequence variation files (e.g. VCF) that were obtained by a sequencing assay (i.e. from the sequenced reads), at least the analysis types 'sequence variation' and 'processed sequencing data' would be expected. Furthermore, depending on the types of analysis, different details may be required (e.g. reference sequence details in a 'sequence alignment' type).",
+            "additionalProperties": false,
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+              "type": "string",
+              "title": "Type of analysis",
+              "description": "Overall type of the analysis. Term chosen from a controlled vocabulary (CV) list. If you cannot find your term in the CV list, please create an issue at our [metadata GitHub repository](https://github.com/EbiEga/ega-metadata-schema) proposing its addition.",
+              "enum": [ "sequence variation", "sequence alignment", "phenotype characterization", "processed sequencing data", "processed array data", "sequence annotation", "sequence assembly" ],
```

Comment:
This enum list is fine for now. In the future, we can switch from a enum list of individual values to providing a higher-level ontology term and allowing any value under that term. For example, under "[sequence analysis objective](http://purl.obolibrary.org/obo/OBI_0200081)".

Reply:
Actually, the terms "processed sequencing data" and "processed array data" don't seem like they belong here. These terms are referencing a type of data, not a type of analysis. 

Maybe add "gene expression"?

Reply:
Added gene expression and moved the types of data elsewhere 👍